app/views/content/train-to-be-a-teacher/if-you-dont-have-a-degree.md

```diff
@@ -35,10 +34,13 @@ Qualifications vary depending on the course. For example, you could get QTS with
 
 [Find out more about the qualifications you need to be a teacher in England](/is-teaching-right-for-me/qualifications-you-need-to-teach).
 
-As part of selecting your degree course, you will need to decide if you want to train to teach at a primary or secondary level. [Learn about deciding who to teach](/is-teaching-right-for-me/who-do-you-want-to-teach).
-
 <div class="inset">
 
 <p>If you're a non-UK citizen who wants to study in England, you can visit the <a href="https://www.ukcisa.org.uk/">UK Council for International Student Affairs</a> for information about studying at an English university.</p>
 
 </div>
+
+## Teacher Degree Apprenticeships (TDAs)
+If you do not have a degree and want to become a teacher you may be eligible to apply for a TDA. This new course allows you work in a school and earn a salary while getting a batchelor's degree and QTS. Applications for both primary and secondary are expected to launch in autumn 2024 and training is due to start in autumn 2025. 
```

Comment:
```suggestion
If you do not have a degree and want to become a teacher you may be eligible to apply for a TDA. This new course allows you work in a school and earn a salary while getting a bachelor's degree and QTS. Applications for both primary and secondary are expected to launch in autumn 2024, with training due to start in autumn 2025. 
```

Reply:
Worth splitting `Applications for....` onto a new para?

Reply:
And do we need `and want to become a teacher`?

Reply:
@gemmadallmandfe I didn't know if it might be good to include it for SEO but I can take it out!

Reply:
we might be better with `become a teacher` traffic going to Steps though?

I think `**teacher** degree **apprenticeships**` will be strong enough re SEO